lib/ldap_fluff/generic_member_service.rb

```diff
@@ -70,7 +70,7 @@ def get_logins(userlist)
 
   def get_login_from_entry(entry)
     [@attr_login, 'uid', 'cn'].each do |attribute|
-      return entry.send(attribute) if entry.respond_to? attribute
+      return entry.send(attribute.map(&:downcase)) if entry.respond_to? attribute
```

Comment:
It seems that my commit doesn't reflect what I tested on the Foreman server. I will check tomorrow.

The intent is to fix membership of groups with. Indeed, it fixes my problem, where samaccountname is upper case (and foreman login is lowercase).